<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arm Test Chips Data Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Settings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>791</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DINPro-Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Input Options</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_Target">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>150</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select Target Graphs:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>185</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Stdcell Yield Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stdcell Vmin Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stdcell Shmoo Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory Yield Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory Vmin Data (EMA)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory Vmin Data (Instance)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory Shmoo Data</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>691</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <strikeout>false</strikeout>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Select Datalog Files:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Process">
        <property name="text">
         <string>PreProcess</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/csv/csv.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LoadCsV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>140</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select Temperature:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Temp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>791</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DINPro-Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Stdcell Advanced Graph Settings</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_StdYield">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string> Std Yield</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_Vol">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Voltage:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Voltage_StdYield">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_StdVmin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Std Vmin</string>
     </property>
     <widget class="QLabel" name="label_Lib">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Library:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Lib">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_StdShmoo">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Std Shmoo</string>
     </property>
     <widget class="QLabel" name="label_Split">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Split:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Split">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>260</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_Split_3">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Range:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="v1std"/>
       </item>
       <item>
        <widget class="QLabel" name="label_Split_4">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>V1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Split_6">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>------</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="v2std"/>
       </item>
       <item>
        <widget class="QLabel" name="label_Split_5">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>V2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>801</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DINPro-Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Memory Advanced Graph Settings</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_MemYield">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Mem Yield</string>
     </property>
     <widget class="QLabel" name="label_Vol_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Voltage:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Voltage_MemYield">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Temp_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ema:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_EmaYield">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_MemShmoo">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>271</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Mem Shmoo</string>
     </property>
     <widget class="QLabel" name="label_Split_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Split:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Split_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>260</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_Split_7">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Range:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="v1Mem"/>
       </item>
       <item>
        <widget class="QLabel" name="label_Split_8">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>V1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Split_9">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>------</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="v2Mem"/>
       </item>
       <item>
        <widget class="QLabel" name="label_Split_10">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>V2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>321</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DINPro-Regular</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Mem Vmin</string>
     </property>
     <widget class="QLabel" name="label_Lib_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Split:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_SplitVmin">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_MemVmin_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DINPro-Regular</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Mem Vmin(Instance Plots)</string>
      </property>
      <widget class="QComboBox" name="comboBox_EmaVmin">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Temp_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ema:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_MemVmin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DINPro-Regular</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Mem Vmin(Ema Plots)</string>
      </property>
      <widget class="QComboBox" name="comboBox_Instance">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Temp_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Instance:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/csv/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionClose_Application"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionClose_Application">
   <property name="text">
    <string>Close Application</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
